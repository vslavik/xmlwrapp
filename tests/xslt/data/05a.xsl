<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
<xsl:output method="html" indent="no" encoding="us-ascii"/>
<xsl:param name="foo" select="'default'"/>
<xsl:template match="/"><HTML><BODY>foo == <xsl:value-of select="$foo"/><xsl:apply-templates/></BODY></HTML></xsl:template>
<xsl:template match="root"><H1>root</H1><xsl:apply-templates/></xsl:template>
<xsl:template match="child"><H3>child</H3><xsl:apply-templates/></xsl:template>
</xsl:stylesheet>
