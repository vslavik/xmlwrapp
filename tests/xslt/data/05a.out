<HTML><BODY>foo == bar<H1>root</H1>
    <H3>child</H3>
    <H3>child</H3>
</BODY></HTML>
